README.md

```diff
@@ -1,39 +1,62 @@
-# Project Template
+# Project Group 72
+* do we have a name for the app? 
```

Comment:
I think we decided on 72Reviews

Reply:
yeah, we can just edit it on the main branch after merging this PR